code/modules/interface/multiContext/context_actions.dm

```diff
@@ -442,6 +442,15 @@
 		else
 			. = ..()
 
+// bodge to support omnitools
+#define OMNI_MODE_PRYING 1
+#define OMNI_MODE_SNIPPING 2
+#define OMNI_MODE_WRENCHING 3
+#define OMNI_MODE_SCREWING 4
+#define OMNI_MODE_PULSING 5
+#define OMNI_MODE_CUTTING 6
+#define OMNI_MODE_WELDING 7
```

Comment:
maybe better to just make these global rather than redefining them here?

Reply:
[dc72de6](https://github.com/goonstation/goonstation/pull/12356/commits/dc72de6950478e20c743160c28bba7d5b31181a7)